core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -122,6 +122,16 @@ public boolean getRequiresConfirmation() {
         return Jenkins.ADMINISTER;
     }
 
+    /**
+     * Whether to show this link on /manage
+     *
+     * @return true if and only if the link should be shown
```

Comment:
```suggestion
     * @return {@code true} iff the link should be shown.
```

Reply:
I used to do that and was sick of code reviews claiming it's a typo. Accessibility > academic superiority.